### PR TITLE
Porfolio mobile size header and headline sections

### PR DESCRIPTION
**Pull request by momo-87**
Add index.html, style.css, README.md, and linters files.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href='https://fonts.googleapis.com/css?family=inter' rel='stylesheet'>
+    <link rel="stylesheet" href="./style.css">
+    <title>Porfolio</title>
+</head>
+<body>
+    <header>
+        <div class="logo-container">
+            <h2>My Logo</h2>
+        </div>
```
Comment (mahbubul14) [inline index.html]: - [ ] The `My Logo` text is on the left and should have a link (a tag). :+1:

File: index.html [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        <div class="open_mobile_menu_icon">
+            <img src="Icons/Menu.svg" alt="">
+        </div>
```
Comment (mahbubul14) [inline index.html]: - [ ] The hamburger button is on the right and should have a nav tag. The `nav` tag is for semantics reasons and the right position for convention and user experience for mobile devices. :+1:

File: index.html [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    </header>
+    <section class="headline">
+        <div class="welcome_text">
+            <h2>Hey There.<br> I'm Christian.<br><span>I’m a software developer</h2></span>  
+        </div>
```
Comment (mahbubul14) [inline index.html]: - [ ] The main title (e.g., "I'm Marianne ...") should be in a `h1` tag. If you want that google (and other search engines) to know about a specific topic, in this case, you, use an h1 tag. Check [this link](https://www.reliablesoft.net/h1-tag/) for more info.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+        <div class="introduction_text">
+            <p class="">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+        </div>
+        <div class="social_media_icons">
+            <div><a href="https://twitter.com/Momo_yde"><img src="Icons/twitter.svg" alt=""></a></div>
+            <div><a href="https://www.linkedin.com/in/christian-momo/"><img src="Icons/Linkedin.svg" alt=""></a></div>
+            <div><a href=""><img src="Icons/medium.svg" alt=""></a></div>
+            <div><a href="https://github.com/Momo-87"><img src="Icons/github.svg" alt=""></a></div>
+            <div><a href=""><img src="Icons/angellist.svg" alt=""></a></div>
+        </div>
```
Comment (mahbubul14) [inline index.html]: - [ ] The list of icons should use a `ul` tag to group them, and `li` per icon; each icon has a link (a tag). Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href. Kindly work on it. :+1:

**Review by mahbubul14 — CHANGES_REQUESTED** [review]
Hi @Momo-87,

Good job so far! :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

# Required Changes ♻️

- [ ] Pull Request should have a short summary describing introduced changes in more detail. Please work on it.
pr


- [ ] Kindly check the inline comments under the review. :arrow_down: 

###  To Highlights
:heavy_check_mark: No linter error.
:heavy_check_mark: Proper Github flow used.

### Optional suggestions

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

* N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by amejid — APPROVED** [review]
Hi @Momo-87, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Linter checks passing :heavy_check_mark: 
- Correct use of Github flow :heavy_check_mark: 
- Good UI design :heavy_check_mark: 
- Good semantic structure :heavy_check_mark: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**